src/_tutorials/web/low-level-html/connect-dart-html.md

```diff
@@ -239,60 +236,89 @@ making all of the classes and functions
-</pre>
+The following `import` statement imports Dart's HTML library,
+which contains key classes and functions for programming the DOM:
+
+```dart
+import 'dart:html';
+```
 
-This program imports Dart's HTML library,
-which contains key classes and functions for programming the DOM.
-Key classes include:
+Some key classes imported include:
 
-| Dart class | Description |
-|---|---|
-| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-html/Node-class.html" target="_blank" rel="noopener">Node</a> | Implements a DOM node. |
-| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-html/Element-class.html" target="_blank" rel="noopener">Element</a> | A subclass of Node; implements a web page element. |
+| Dart class                                                                                                                               | Description                                               |
+|------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------|
+| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-html/Node-class.html" target="_blank" rel="noopener">Node</a>         | Implements a DOM node.                                    |
+| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-html/Element-class.html" target="_blank" rel="noopener">Element</a>   | A subclass of Node; implements a web page element.        |
 | <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-html/Document-class.html" target="_blank" rel="noopener">Document</a> | Another subclass of Node; implements the document object. |
 {: .table}
 
-The Dart core library contains another useful class:
+The dart:core library, which is automatically imported, 
+contains many other useful classes, such as:
 <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/List-class.html" target="_blank" rel="noopener">List,</a>
 a parameterized class that can specify the type of its members.
-An instance of Element keeps its list of child Elements
-in a List\<Element>.
+An instance of `Element` keeps its list of child elements
+in a `List<Element>`.
 
 ### Using the querySelector() function
 
-This app's main() function contains a single
+This app's `main()` function contains a single
 line of code that is a little like a run-on sentence
 with multiple things happening one after another.
 Let's deconstruct it.
 
-querySelector() is a top-level function provided by the Dart HTML library
-that gets an Element object from the DOM.
+`querySelector()` is a top-level function provided by the Dart HTML library
+that gets an `Element?` object from the DOM matching the passed in specifier.
```

Comment:
I don't think adding more words to this sentence helps... it's already pretty long, and those details are covered below.

```suggestion
that gets an `Element?` object from the DOM.
```